node/node.go

```diff
@@ -650,12 +652,10 @@ func (node *Node) Start() error {
 		msgChan := make(chan validated, MsgChanBuffer)
 
 		go func() {
-
-			for msg := range msgChan {
-				// should not take more than 3 seconds to process one message
-				ctx, cancel := context.WithTimeout(context.Background(), 3*time.Second)
-				msg := msg
-
+			// should not take more than 5 seconds to process one message
+			ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
```

Comment:
Does it work to have a single context object to use for all messages?

Reply:
fixed.